src/main/java/HW9/FileNavigator.java

```diff
@@ -0,0 +1,62 @@
+package HW9;
+
+import java.util.*;
+
+public class FileNavigator {
+    private final Map<String, List<FileData>> records = new HashMap<>();
+    public void add(String path, String nameOfFile, long sizeOfFile){
+        FileData fileData = new FileData(path, nameOfFile,sizeOfFile);
+
+        if(records.containsKey(path)){
+            List<FileData> fileDescription = records.get(path);
+            fileDescription.add(fileData);
+        } else {
+            List<FileData> fileDescription = new ArrayList<>();
+            fileDescription.add(fileData);
+            records.put(path, fileDescription);
+        }
+    }
+
+    public List<FileData> find(String path){
+        if(records.containsKey(path)){
+            return records.get(path);
+        }
+        return null;
```

Comment:
можна в один рядок: 
```suggestion
        return records.get(path);
```
те саме буде.